src/components/video/VideoCard.vue

```diff
@@ -360,7 +361,7 @@ export default {
             type: Number,
             default: 1,
         },
-        active: {
+        active: { // TODO: is this always false?
```

Comment:
no, it gets highlighted when it's active in a list

Reply:
Ah i think we were using active-index on a virtual video card list instead

Reply:
Should I just remove the TODOs?

Reply:
it's fine i think, or just update it to reflect this fact lol